content/en/examples/application/job/job-sidecar.yaml

```diff
@@ -16,7 +16,7 @@ spec:
         - name: logshipper
           image: alpine:latest
           restartPolicy: Always
-          command: ['sh', '-c', 'tail /opt/logs.txt']
+          command: ['sh', '-c', 'touch /opt/logs.txt && tail --follow=file /opt/logs.txt']
```

Comment:
alpine uses busybox which doesn't understand --follow, please revert to `-f`

Reply:
Thanks for your reminder, updated done.

Reply:
also why did you need the `touch`? if the file isn't there (race condition) maybe you can try with `-F` which stands for `Same as -f, but keep retrying`

Reply:
it's make sense, honestly I have never use the `-F` for tail.
test with `-F` in my local all looks fine.